k8s/k8s.go

```diff
@@ -4487,6 +4493,29 @@ func (k *k8sOps) UpdateBackupLocation(backupLocation *v1alpha1.BackupLocation) (
 	return k.storkClient.Stork().BackupLocations(backupLocation.Namespace).Update(backupLocation)
 }
 
+func (k *k8sOps) ValidateBackupLocation(name, namespace string, timeout, retryInterval time.Duration) error {
+	t := func() (interface{}, bool, error) {
+		if err := k.initK8sClient(); err != nil {
+			return "", true, err
+		}
+
+		resp, err := k.GetBackupLocation(name, namespace)
+		if err != nil {
+			return "", true, &ErrFailedToValidateCustomSpec{
+				Name:  name,
+				Cause: fmt.Sprintf("BackupLocation %v: %v failed . Error: %v", name, namespace, err),
```

Comment:
Name is already being passed into the error, don't need to print it again.

Reply:
Fixed.